.github/workflows/ci.yml

```diff
@@ -58,4 +58,4 @@ jobs:
 
     - name: Test
       run: |
-        ./script/test -v
+        uv run ./script/test -v
```

Comment:
Someday, we may even not need a custom script and we can launch pytest directly :D